### PR TITLE
Freedom of owospeech - Adds the "uwuspeech" mutation

### PR DESCRIPTION
**Pull request by Lauwma**
## About The Pull Request

Adds a new mutation called "uwu". This mutation replaces your R and L with "w". This is a dormant mutation in felinids DNA, you cannot find it in any other species.

## Why It's Good For The Game

Wouldn't it be hilarious if a traitor geneticist decided to blast the whole crew with "uwu" mutators ? It adds a little bit of soft grief to the game (you can still revert it with mutadone if you don't like that mutation). Admins can also use this if they feel like doing a little trolling.

## Changelog
:cl:
add: The geneticist discovered a long forgotten mutation called "uwuspeech" on felinids specimens. Be caweful you may stawt to- oh no...
/:cl:

### CODE REVIEW

**Comment by Lauwma** [comment]
Seems like my pull messed with the logs, I'm not an experienced gitter so I don't really know how to fix this

**Comment by Iamgoofball** [comment]
This is awful. Please do not fucking merge this, for the love of god. Words cannot express how awful this is.

**Comment by Rowelll** [comment]
lmao

**Comment by Nexofas** [comment]
speedmerge

**Comment by Fikou** [comment]
https://tenor.com/view/the-mask-shocked-jim-carrey-awooga-gif-5428977

**Comment by Fikou** [comment]
https://tenor.com/view/the-mask-shocked-jim-carrey-awooga-gif-5428977

File: code/game/objects/items/dna_injector.dm [hunk 1]
```diff
@@ -444,6 +444,14 @@
 	name = "\improper DNA injector (Anti-Webbing)"
 	remove_mutations = list(SPIDER_WEB)
 
+/obj/item/dnainjector/uwu
+	name = "\improper DNA injector (Webbing)"
```
Comment (JJRcop) [inline code/game/objects/items/dna_injector.dm:448]: ???

File: code/game/objects/items/dna_injector.dm [hunk 2]
```diff
@@ -444,6 +444,14 @@
...
+	add_mutations = list(UWU)
+
+/obj/item/dnainjector/antiuwu
+	name = "\improper DNA injector (Webbing)"
```
Comment (JJRcop) [inline code/game/objects/items/dna_injector.dm:452]: ???

**Review by JJRcop — COMMENTED** [review]
uwebbing

**Comment by JJRcop** [comment]
Diesn't seem to handle capital R or L

**Comment by JJRcop** [comment]
also it should add u before every w, and wu after every u
This uway youwu tawk wike this..

File: code/game/objects/items/dna_injector.dm (hunk 1 above)
Comment (Lauwma) [reply code/game/objects/items/dna_injector.dm]: Thanks for pointing that out !

File: code/datums/mutations/speech.dm [hunk 3]
```diff
@@ -240,3 +240,31 @@
 		message = "[chosen_starting] [message]"
 
 		speech_args[SPEECH_MESSAGE] = message
+
+/datum/mutation/human/uwu
+	name = "uwuspeech"
```
Comment (Fikou) [inline code/datums/mutations/speech.dm:245]: why not capitalized

**Comment by Lauwma** [comment]
> Doesn't seem to handle capital R or L

The function doing it does handle it I tested it

**Comment by Lauwma** [comment]
Waiting for maint approval as advised.

**Comment by exymian** [comment]
say sike rn

**Comment by 81Denton** [comment]
cringe

**Comment by Burgermancoding** [comment]
1984

**Comment by Lauwma** [comment]
In my biased opinion this addition opens opportunities to escalated pranks when people break in genetics without asking you. I have experienced few bwoinks when I injected people with blind mutator into assistants breaking in genetics because it was a too harsh punishements. Injecting Swedish, Chav or Medieval talk feels like not being a punishment at all. Though, uwuspeeching feels like a correct escalation because you get to see the person speaking like a retard without griefing him too much since he can just stays silent and go fetch mutadone. 
Sure this PR doesn't add a strong powergamey mutation but I do think it has its place in a goof game like SS13. After all, a lot of features in the game doesn't have a practical use (French Beret, syndicate balloon, ect...) but still add a lot of flavors and funs to the rounds.

**Comment by Nexofas** [comment]
> cringe
> 
> edit: To elaborate, I think that this feature does not add anything meaningful or interesting in the game that doesn't exist already. It also does not fit the innate felinid language and is cringy as hell.

There is a "swedish" mutation". This mutation is almost the same but with a felinid touch. It adds a funny mutation just like other funny mutations and its felinid dormant. It cant break the game can it?

**Comment by Watermelon914** [comment]
bad design from the past isn't really good justification for bad design in the future.

**Comment by Nexofas** [comment]
> bad design from the past isn't really good justification for bad design in the future.

There is a difference between "bad design" and "fun additions"
Why would this pr fall in said "bad design" category?

**Comment by interestingusernam3** [comment]
> Iamgoofball closed this 6 hours ago 

https://www.youtube.com/watch?v=6EfhAFA2yFE

**Comment by 81Denton** [comment]
> Why would this pr fall in said "bad design" category?

image

**Comment by Lauwma** [comment]
What about the constructives things I said above ? If I feel like some people would enjoy such addition to bug others and that having this mutation don't do nothing for those who won't use it, why would you not accept it ?

Plus I don't see why this mutation is problematic lorewise. If you want an explanation why felinids would speak like that, let's say this mutation makes your tongue sticky like cat's one, making it harder to pronounce "R" and "L". I just didn't want to name it "cat tongue" "fluffy tongue" or anything tongue related because it feels like it would be creepy.

Other that "this is cringe" I've yet to understand an explaination to why this was closed. Just saying this won't help me contribute in the future because I won't know what to improve.
I read the guidelines, tested the code, standardized it. I tried to make something that looked funny for me and others knowing that others wouldn't understand the joke. But ev if you don't like it I've to understand another reason to why it was refused, a reason that would make me feel that decision is fair.

**Comment by interestingusernam3** [comment]
@Lauwma Not a maintainer, but I think
1. Violation of https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#banned-content
2. Reference style. Here's two comments by Rohesie from #59306:
`
Can you be somewhat less in-your-face with the OOC references?

The game is built upon references and jokes, but a certain degree of subtlety or at least deniability help build a better ambience.
`

`It's still full of direct OOC content.
Remove the based cringe epic gaming etc dumb stuff and try to be a little bit more subtle with references.

Look at other meta references for examples on how to do this, such as the several soft drinks (robust, pwr game, orange/lemon-lime), cigarettes, beams crossing, BoH inside a BoH and whatnot.`
Also here's a relevant but probably useless hackmd document: https://hackmd.io/V911cDOPRhqAsoCQP1aUrA
3. This is reddit. Maintainers hate reddit.
4. Maintainers dislike uwuspeak.
5. Don't feed the furries.
6. Don't feed the felinid mains.
7. Don't feed the ERPers.
8. This is bad.
9. This does not stylistically fit the /tg/station codebase in general. 
10. You behave like MystLeissa, but without the comical egregiousness.
11. This is fetish content.
12. ~~This violates the github TOS.~~
12. ~~This is going to kill the game~~
12. This looks like a feature from a game made by a furry.
13. You didn't think this through.
14. oranges won't let you do this.
15. I think this feature could really assist some of our manuel players in roleplaying the true depth of their commitments to each other.
16. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
17. Der Mensch ist von Natur böse. Er tut das Gute nicht aus Neigung, sondern aus Sympathie und Ehre.
18. ░░░░░▄▄▄▄▀▀▀▀▀▀▀▀▄▄▄▄▄▄░░░░░░░
░░░░░█░░░░▒▒▒▒▒▒▒▒▒▒▒▒░░▀▀▄░░░░
░░░░█░░░▒▒▒▒▒▒░░░░░░░░▒▒▒░░█░░░
░░░█░░░░░░▄██▀▄▄░░░░░▄▄▄░░░░█░░
░▄▀▒▄▄▄▒░█▀▀▀▀▄▄█░░░██▄▄█░░░░█░
█░▒█▒▄░▀▄▄▄▀░░░░░░░░█░░░▒▒▒▒▒░█
█░▒█░█▀▄▄░░░░░█▀░░░░▀▄░░▄▀▀▀▄▒█
░█░▀▄░█▄░█▀▄▄░▀░▀▀░▄▄▀░░░░█░░█░
░░█░░░▀▄▀█▄▄░█▀▀▀▄▄▄▄▀▀█▀██░█░░
░░░█░░░░██░░▀█▄▄▄█▄▄█▄████░█░░░
░░░░█░░░░▀▀▄░█░░░█░█▀██████░█░░
░░░░░▀▄░░░░░▀▀▄▄▄█▄█▄█▄█▄▀░░█░░
░░░░░░░▀▄▄░▒▒▒▒░░░░░░░░░░▒░░░█░
░░░░░░░░░░▀▀▄▄░▒▒▒▒▒▒▒▒▒▒░░░░█░
░░░░░░░░░░░░░░▀▄▄▄▄▄░░░░░░░░█░░
19. why
20. 
23. I just downvoted your comment.

# FAQ

## What does this mean?

The amount of karma (points) on your comment and Reddit account has decreased by one.

## Why did you do this?

There are several reasons I may deem a comment to be unworthy of positive or neutral karma. These include, but are not limited to:

• ⁠Rudeness towards other Redditors, • ⁠Spreading incorrect information, • ⁠Sarcasm not correctly flagged with a /s.

## Am I banned from the Reddit?

No - not yet. But you should refrain from making comments like this in the future. Otherwise I will be forced to issue an additional downvote, which may put your commenting and posting privileges in jeopardy.

## I don't believe my comment deserved a downvote. Can you un-downvote it?

Sure, mistakes happen. But only in exceedingly rare circumstances will I undo a downvote. If you would like to issue an appeal, shoot me a private message explaining what I got wrong. I tend to respond to Reddit PMs within several minutes. Do note, however, that over 99.9% of downvote appeals are rejected, and yours is likely no exception.

## How can I prevent this from happening in the future?

Accept the downvote and move on. But learn from this mistake: your behavior will not be tolerated on Reddit.com. I will continue to issue downvotes until you improve your conduct. Remember: Reddit is privilege, not a right.

**Comment by interestingusernam3** [comment]
It's not spam.

**Pull request closed without merging**